src/sonic-device-data/tests/config_checker

```diff
@@ -34,6 +34,7 @@ def check_file(file_name):
                 p = re.sub(r"\.[0-9]+$", '', p)
                 # Remove trailing port name
                 p = re.sub(r"_[cxg]e(\d+)?$", '', p)
+                p = re.sub(r"\{.*\}", '', p)
```

Comment:
can you add a comment for this? what does this line try to do?

Reply:
we comments for all pre-processing here.

Reply:
It's to remove the TH3 trailing port id
e.g.
serdes_core_rx_polarity_flip_physical{137}

Reply:
Please check commit 2 for updates, I'm not familiar with this GUI, I have no idea why the change is not showing up in this conversation...